jicofo/rootfs/healthcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl --fail-with-body -v localhost:8888/about/health
```

Comment:
Why the verbose flag? Also, any reason not to use 127.0.0.1 and skip a lookup?

Reply:
good point, it is removed :+1: